regex/noncap-hours.regex

```diff
@@ -1 +1 @@
-(?:[0-9]{4}[AP]M)?-([0-9]{4}[AP]M)?
+(?:[0-9]{4}[AP]M)?-?(?:[0-9]{4}[AP]M)?
```

Comment:
Why is the dash that separates the two sections optional? Is `1130AM0200PM` a valid time?